frontend/src/components/Button.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import { NavLink } from 'react-router-dom'
+
+export function Button({ children }) {
+  return (
+    <LinkStyled to="result" onClick={countTotalCheckedBoxes}>
+      {children}
+    </LinkStyled>
+  )
+}
+
+export function countTotalCheckedBoxes() {
+  return document.querySelectorAll('input:checked').length
+}
+
+// is it possible to see color change on-click (activeClassName didn't work), like you see when hovering in deskopt display?
+const LinkStyled = styled(NavLink)`
+  width: 300px;
+  height: 48px;
+  border-radius: 8px;
+  box-shadow: -4px 5px 8px 0 rgba(67, 86, 100, 0.12),
+    -1px 2px 4px 0 rgba(67, 86, 100, 0.1);
+  background: var(--secondary);
+  margin: 40px auto 80px auto;
+  color: var(--primary-light-8);
+  font-size: 18px;
+  font-weight: 700;
+  text-align: center;
+  line-height: 48px;
+  text-decoration: none;
+
+  &:hover {
+    background: var(--primary-light);
+    color: var(--secondary);
+  }
+`
```

Comment:
Brauchst du den Kommentar noch? Ansonsten lieber rausnehmen, bevor du alles in den Master einfügst :)
Und was ich vorhin schon gesagt habe - du nutzt querySelectorAll in React, dafür solltest du dir noch was anderes überlegen :)

Reply:
Kommentar schreib ich mir raus, danke :) Und: document.querySelectorAll konnte ich auch ersetzen.